### ISSUE TITLE
Tree component is have error in typescript & react project?

### ISSUE THREAD

**Issue by luojinghui** [diogofcunha/react-virtualized-tree#57]
```jsx
import Tree from 'react-virtualized-tree';
import { renderers } from 'react-virtualized-tree';
import 'react-virtualized/styles.css';
const { Expandable } = renderers;

	constructor(props: IMainProps) {
		super(props);
		this.state = {
			nodes: Nodes
		};
	}
```
```jsx
      <Tree nodes={this.state.nodes} onChange={this.handleChange}>
            {({ node, ...rest }) => (
                <Expandable node={node} {...rest}>
                    {node.name}
                </Expandable>
            )}
        </Tree>
```
```jsx
	public handleChange = (nodes: any) => {
		this.setState({ nodes });
	};
```

![image](https://user-images.githubusercontent.com/12367406/44500882-634fbd00-a6bd-11e8-9dab-1613dac6b696.png)

There are errors in the picture in typescript , but my data is created using the methods in your demo. I don't know why I will report an error. Please help me.

**Comment by diogofcunha** [comment]
Looks like the types are wrong, will publish a fix

**Comment by diogofcunha** [comment]
Fixed in version 2.3.1. Thanks for reporting
